package.json

```diff
@@ -4,7 +4,7 @@
     "registry": "https://npm.pkg.github.com"
   },
   "repository": "git://github.com/contacto-io/contacto-web-app",
-  "version": "0.1.44",
```

Comment:
change it to 46. I have already published with 45

Reply:
done